apps/time/src/widgets/time-table/model/hooks/useLectureByParams.ts

```diff
@@ -0,0 +1,43 @@
+import { useQuery } from '@tanstack/react-query';
+
+import { getLectureByParams } from '@/widgets/time-table/api';
+import {
+  getFormattedLectureList,
+  timeTableQueryKeys,
+  useTimeTableParams,
+} from '@/widgets/time-table/model';
+import { useRouter } from 'next/navigation';
+
+export function useLectureByParams(idList: number[]) {
+  const { searchParamsAction } = useTimeTableParams();
+  const { dayStatus } = useTimeTableParams();
+  const router = useRouter();
+  const { data, isError } = useQuery({
+    queryKey: timeTableQueryKeys.getLectureByParams(idList),
+    queryFn: idList.length
+      ? () => getLectureByParams(idList)
+      : () => Promise.resolve(null),
+    enabled: !!idList.length,
```

Comment:
```suggestion
  const { data, isError } = useQuery({
    queryKey: timeTableQueryKeys.getLectureByParams(idList),
    queryFn: () => getLectureByParams(idList)
    enabled: !!idList.length,
```
'enabled'로 이미 있을 경우 해당 쿼리가 실행되도록 되어 있어요, 'null'을 사용해야 하는 케이스가 있나요??

Reply:
어 그렇네요 `enabled`가 이미 되어있어서 해당 케이스가 필요하지 않겠네요!